packages/tools/devtools/devtools/src/index.ts

```diff
@@ -187,6 +189,14 @@ function mapContainerProps(
 	};
 }
 
+/**
+ * Gets the Devtools singleton if it has been initialized, otherwise returns `undefined`.
+ * @beta
+ */
+export function tryGetFluidDevtools(): IFluidDevtools | undefined {
```

Comment:
You'll also need to add this to the package's root exports (index.ts), or Loop won't be able to import it.

Reply:
Sorry, I missed where this function was declared. We should add this to `devtools-core` (probably right next to `initializeFluidDevtools`.

I'm not sure we need to add anything to this package. We can always add a wrapper for it (like `initializeDevtools` above) as needed in the future. But Loop should only depend on `devtools-core` and not `devtools`, so we can wait on that.